### ISSUE TITLE
EASY API: implement `ngrok` or `localtunnel` flag to expose api for easy remote access

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2314]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "minor"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.24.0](https://github.com/open-wa/wa-automate-nodejs/compare/4.23.18...4.24.0) (2021-11-24)

- Release 4.24.0 [`300acda`](https://github.com/open-wa/wa-automate-nodejs/commit/300acda5f091da26a7e001121702b20c917cc3c4)
- ✨ CLI: Expose EASY API with `--tunnel` #2314 [`bc60394`](https://github.com/open-wa/wa-automate-nodejs/commit/bc6039495ae791234c1b2b37a714da7c4ebe722b)
- ♻️  replaced `glob` with `tiny-glob`. Fixed glob check path #2317 [`9caa93f`](https://github.com/open-wa/wa-automate-nodejs/commit/9caa93f8e7aefd32e973329cfac543dc9aa71d2b)
- 🔊  improved client method error logs #2316 [`59a3130`](https://github.com/open-wa/wa-automate-nodejs/commit/59a31305f7d60215a76bcf1eb034cd5e00492220)
- ✨ automatically detect MD session when env var `AUTO_MD=TRUE` #2313 [`cd95aed`](https://github.com/open-wa/wa-automate-nodejs/commit/cd95aed7792e5ffe0088fecf1beb3d815bed1122)
- 🔧  double qrTimeout if `multiDevice` is on #2315 [`e499ccd`](https://github.com/open-wa/wa-automate-nodejs/commit/e499ccdd3339d33285eb2ec446a8832a066e2992)
- 🔧  update useragent [`292591f`](https://github.com/open-wa/wa-automate-nodejs/commit/292591f5746fba5b74a5219983e9e7047a623d06)
- 🥚  patches update [`f0bf4bc`](https://github.com/open-wa/wa-automate-nodejs/commit/f0bf4bc404030c6180225bc44a38537807111086)
- updated types-only package [`fa4afb3`](https://github.com/open-wa/wa-automate-nodejs/commit/fa4afb3099b1bff489af36f6be09b3e7ef18f109)
